stream-chat-android-ui-components/src/main/res/layout/stream_ui_channel_list_item_background_view.xml

```diff
@@ -3,7 +3,7 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
     android:layout_height="@dimen/stream_ui_channel_list_item_height"
-    android:background="@color/stream_ui_gray_legacy"
+    android:background="?android:attr/colorBackground"
```

Comment:
I did't see difference between `@color/stream_ui_gray_legacy` and `?android:attr/colorBackground`so I use the second because it provides  support for Dark/Light theme out of the box. I can deal with this colour in a programatically way if there's the need.